doc/source/library/simulator/config.rst

```diff
@@ -68,6 +68,9 @@ The entry “framer” allows the following values:
 - “tls” to use :class:`pymodbus.framer.ModbusTlsFramer`,
 - “socket” to use :class:`pymodbus.framer.ModbusSocketFramer`.
 
+Optional entry "unit_address" will enable address-specific server addressing. If
```

Comment:
The modbus term is device_id or slave (whic( we use throughout), please change.

Reply:
Changed to device_id.

(I'm grateful for your comment. The preferred terminology is sometimes not easy to deduce in cases like this 😃 )